tests/rptest/services/redpanda.py

```diff
@@ -2891,14 +2891,18 @@ def remove_local_data(self, node):
 
     def redpanda_pid(self, node):
         try:
-            cmd = "pgrep --exact redpanda"
-
-            for p in node.account.ssh_capture(cmd,
-                                              allow_fail=True,
-                                              callback=int,
-                                              timeout_sec=10):
-                return p
-
+            cmd = "pgrep --list-full --exact redpanda"
+            for line in node.account.ssh_capture(cmd,
+                                                 allow_fail=True,
+                                                 callback=int,
```

Comment:
I wonder how it works with `callback=int` after the command was changed to print the full cmdline.

Reply:
🤦 fixed.